JPS_Ontology/ontology/ontobms/OntoBMS.csv

```diff
@@ -1,88 +1,51 @@
+HighPlumeFan,Class,ExhaustFan,IS-A,,,,A type of exhaust fan that dilute and disperse potentially hazardous exhaust or fumes quickly and safely.,https://www.theworldavatar.com/kg/ontobms/,High plume fan
+AlarmState,Class,https://saref.etsi.org/core/MultiLevelState,IS-A,,,,An alert status that is triggered due to an abnormal condition or change in condition.,https://www.theworldavatar.com/kg/ontobms/,Alarm state
+DamperState,Class,https://saref.etsi.org/core/MultiLevelState,IS-A,,,,The current state of a damper.,https://www.theworldavatar.com/kg/ontobms/,Damper state
+SwitchState,Class,https://saref.etsi.org/core/MultiLevelState,IS-A,,,,The current state of a switch.,https://www.theworldavatar.com/kg/ontobms/,Switch state
+LowLevelState,Class,https://saref.etsi.org/core/State,IS-A,,,,The current state of a device at a low level.,https://www.theworldavatar.com/kg/ontobms/,Low level state
+MediumLevelState,Class,https://saref.etsi.org/core/State,IS-A,,,,The current state of a device at a medium level.,https://www.theworldavatar.com/kg/ontobms/,Medium level state
+HighLevelState,Class,https://saref.etsi.org/core/State,IS-A,,,,The current state of a device at a high level.,https://www.theworldavatar.com/kg/ontobms/,High level state
+ElectricalPower-AC,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Power,IS-A,,,,Alternating Current (AC) electrical power.,https://www.theworldavatar.com/kg/ontobms/,Electrical power (AC)
+ElectricalPower-DC,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Power,IS-A,,,,Direct Current (DC) electrical power.,https://www.theworldavatar.com/kg/ontobms/,Electrical power (DC)
+HeatDuty,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/HeatFlowRate,IS-A,,,,The amount of heat needed to transfer from a hot side to the cold side over a unit of time.,https://www.theworldavatar.com/kg/ontobms/,Heat duty
+ConvectiveTransportRate,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/MassFlow,IS-A,,,,A Convective Transport Rate in the most general terms refers to the movement of molecules within fluid phases which is the sum of advective and diffusive transport.,https://www.theworldavatar.com/kg/ontobms/,Convective transport rate
+DamperFeedback,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Pressure,IS-A,,,,A signal (pressure based) that that can be used to determine the exact position of a damper.,https://www.theworldavatar.com/kg/ontobms/,Damper feedback
+DamperControl,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Pressure,IS-A,,,,A pressure based measurement used to control the volume of air that flows through your HVAC system by providing measurements to control the damper position.,https://www.theworldavatar.com/kg/ontobms/,Damper control
+SashOpening,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Percentage,IS-A,,,,The amount of opening for a sash. ,https://www.theworldavatar.com/kg/ontobms/,Sash opening
```

Comment:
A sash opening itself _is_ not a percentage - only the extent to which it is opened. So, defining SashOpening as subclass of Percentage is not appropriate, I believe. How about something like SashOpening hasOpenPercentage OpenPercentage, and then OpenPercentage rdfs:subClassOf Percentage? Or perhaps you're looking for FumeHood hasSashOpenPercentage OpenPercentage, with OpenPercentage rdfs:subClassOf Percentage?